pom.xml

```diff
@@ -352,7 +353,7 @@
             <dependency>
                 <groupId>org.apache.struts</groupId>
                 <artifactId>struts-annotations</artifactId>
-                <version>1.0.5</version>
+                <version>1.0.6-SNAPSHOT</version>
```

Comment:
Now I get it, we must first update struts-annotations project, release it and then we can use it here